v2/tools/generator/internal/codegen/pipeline/stage.go

```diff
@@ -131,9 +133,18 @@ func (stage *Stage) RequiresPrerequisiteStages(prerequisites ...string) {
 }
 
 // RequiresPostrequisiteStages declares which stages must be executed after this one has completed
-// This is not completely isomorphic with RequiresPrerequisiteStages as there may be supporting stages that are
+// Use postrequisites when it is necessary for that later stage to act on the results of this one.
+//
+// For example, InjectJsonSerializationTests creates round trip serialization tests for any object types that have
+// properties. It's not correct to give InjectJsonSerializationTests a prerequisite on every earlier stage that creates
+// new object types becauses it isn't concerned with where those object came from. However, but those earlier stages DO
```

Comment:
Minor/wording: `However, but` on this line is phrased strangely? maybe just:
```suggestion
// new object types becauses it isn't concerned with where those object came from. However, those earlier stages DO
```

Reply:
Updated.